mentat/code_file_manager.py

```diff
@@ -138,7 +139,11 @@ def _set_file_paths(
             print("Exiting...")
             exit()
 
-        excluded_files = _abs_file_paths_from_list(exclude_paths, check_for_text=False)
+        excluded_files, excluded_files_from_dir = _abs_file_paths_from_list(
+            exclude_paths, check_for_text=False
+        )
+        # excluded paths should have already been expanded
+        assert not excluded_files_from_dir
```

Comment:
? Why is this? What if the user tries to --exclude directory?

Reply:
Oh oops. You want to fix?

Reply:
Yeah i'll get it